### PR TITLE
 Break the B2B sample application into two separate applications.

### PR DESCRIPTION
**Pull request by Achintha444**
## Purpose
The main purpose of this PR is to break the B2B sample application into two separate applications `business-app` and `business-admin-app`.

## Goals
The main goal of this procedure is to properly demonstrate the full capabilities of our B2B feature.

## Approach
The plan for this procedure is to use [Nx](https://nx.dev/) to change the repo into a monolithic repo with two separate applications. Because it will allow the developer to easily manage the code base.

### CODE REVIEW

File: b2b-sample/.eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+                "*.spec.js",
+                "*.spec.jsx"
+            ],
+            "env": {
+                "jest": true
+            },
+            "rules": {}
+        }
+    ]
+}
```
Comment (thivi) [inline b2b-sample/.eslintrc.json:71]: Add a new line.

File: b2b-sample/.gitignore [hunk 2]
```diff
@@ -33,3 +33,6 @@ yarn-error.log*
 # vercel
 .vercel
 .idea/**
+
+# Next.js
+.next
```
Comment (thivi) [inline b2b-sample/.gitignore:38]: Add a new line.

**Comment by thivi** [comment]
Can we use `@wso2is` as the package scope name?

**Comment by Achintha444** [comment]
> Can we use `@wso2is` as the package scope name?

Since this task requires more time, will track it with another PR

**Review by thivi — APPROVED** [review]

File: b2b-sample/apps/business-admin-app/components/sections/home.tsx [hunk 3]
```diff
@@ -16,36 +16,34 @@
...
-import DashboardSectionComponent from "./sections/dashboardSection/dashboardSectionComponent";
 import IdpSectionComponent from "./sections/settingsSection/idpSection/idpSectionComponent";
 import ManageUserSectionComponent from "./sections/settingsSection/manageUserSection/manageUserSectionComponent";
 import RoleManagementSectionComponent from
     "./sections/settingsSection/roleManagementSection/roleManagementSectionComponent";
+import sideNavData from "../../../../libs/business-admin-app/ui/ui-assets/src/lib/data/sideNav.json";
 import Custom500 from "../../pages/500";
-import styles from "../../styles/Settings.module.css";
-import SignOutModal from "../common/signOutModal";
+
```
Comment (lashinijay) [inline b2b-sample/apps/business-admin-app/components/sections/home.tsx:30]: unwanted new line. Please check other places also.

File: b2b-sample/apps/business-admin-app/components/sections/sections/settingsSection/idpSection/otherComponents/idpDetailsSections/settings.tsx [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+                                                </Button>
+                                            </ButtonToolbar>
+                                        </FormSuite.Group>
+                                    </div>
+                                </FormSuite>
+                            ) }
+                        />)
+                        : null
+                }
+
```
Comment (ThaminduR) [inline b2b-sample/apps/business-admin-app/components/sections/sections/settingsSection/idpSection/otherComponents/idpDetailsSections/settings.tsx:135]: do we need this newline?

File: b2b-sample/apps/business-admin-app/styles/Home.module.css [hunk 5]
```diff
@@ -0,0 +1,111 @@
...
+    }
+
+    .code {
+        background: #111;
+    }
+
+    .logo img {
+        filter: invert(1);
+    }
+}
```
Comment (ThaminduR) [inline b2b-sample/apps/business-admin-app/styles/Home.module.css:111]: let's add a new line before the page end.

File: b2b-sample/apps/business-admin-app/components/sections/home.tsx (hunk 3 above)
Comment (Achintha444) [reply b2b-sample/apps/business-admin-app/components/sections/home.tsx:30]: Fixed with https://github.com/wso2/samples-is/pull/416/commits/3e0450172639aa334cdc6716a7e8640db58efa19

File: b2b-sample/apps/business-admin-app/styles/Home.module.css (hunk 5 above)
Comment (Achintha444) [reply b2b-sample/apps/business-admin-app/styles/Home.module.css:111]: Fixed with https://github.com/wso2/samples-is/pull/416/commits/3e0450172639aa334cdc6716a7e8640db58efa19

File: b2b-sample/apps/business-admin-app/components/sections/sections/settingsSection/idpSection/otherComponents/idpDetailsSections/settings.tsx (hunk 4 above)
Comment (Achintha444) [reply b2b-sample/apps/business-admin-app/components/sections/sections/settingsSection/idpSection/otherComponents/idpDetailsSections/settings.tsx:135]: Fixed with https://github.com/wso2/samples-is/pull/416/commits/3e0450172639aa334cdc6716a7e8640db58efa19

**Pull request merged**